src/commonMain/kotlin/exchange.dydx.abacus/validator/InputValidator.kt

```diff
@@ -29,7 +29,6 @@ internal class InputValidator(
         "MARKET_ORDER_NOT_ENOUGH_LIQUIDITY" to 1030,
         "MARKET_ORDER_ERROR_INDEX_PRICE_SLIPPAGE" to 1031,
         "MARKET_ORDER_ERROR_ORDERBOOK_SLIPPAGE" to 1032,
-        "MARKET_ORDER_PRICE_IMPACT_AT_MAX_LEVERAGE" to 1033,
```

Comment:
do we also want to remove `INVALID_LARGE_POSITION_LEVERAGE`, `INVALID_NEW_POSITION_LEVERAGE` and `MARKET_ORDER_CLOSE_TO_MAX_LEVERAGE` below?